kernel/univ.mli

```diff
@@ -30,17 +30,13 @@ sig
       definition or global. *)
 
   val set : t
-  val prop : t
-  val sprop : t
-  (** The set and prop universe levels. *)
+  (** The Set universe level. *)
 
   val is_small : t -> bool
-  (** Is the universe set or prop? *)
+  (** Is the universe Set? *)
```

Comment:
Is it still useful to have this function?

Reply:
This will go away at the next cleanup, but I didn't want to introduce (too many) API incompatibilities in this PR to get a full bench.